src/lithium/__init__.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# coding=utf-8
+# flake8: noqa
+# pylint: disable=missing-docstring
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import lithium.interestingness.crashes
+# import lithium.interestingness.diff_test  # diff_test is not used outside of Lithium
+import lithium.interestingness.hangs
+import lithium.interestingness.outputs
+import lithium.interestingness.range
+import lithium.interestingness.timed_run
+import lithium.interestingness.utils
+from lithium.reducer import *  # pylint: disable=wildcard-import
```

Comment:
Why have you used `lithium.reducer` here instead of just `from .reducer ...`?

Reply:
I guess I was trying to sync on all the absolute imports stuff. This can be reverted now that the interestingness imports are to be moved to `interestingness/__init__.py`